### DISCUSSION TITLE
Smoke Detector Template difference

### DISCUSSION THREAD

**Discussion by elajoie** [arendst/Tasmota#11429] [Q&A] [answered]
Hi,

When I try to apply a template to my [smoke detector](https://templates.blakadder.com/visortech_RWM-200_app.html) like this one:

`{"NAME":"VisorTech RWM-200","GPIO":[0,107,0,108,0,0,0,0,0,0,0,0,0],"FLAG":0,"BASE":54}`

and I go back to edit I see this:

`{"NAME":"VisorTech RWM-200","GPIO":[0,2272,0,2304,0,0,0,0,0,0,0,0,0,0],"FLAG":0,"BASE":54}`

Any idea what I might be doing wrong? The 107 and 108 are replace and one more 0 added to the end.

Thanks,
Eric

**Comment by sfromis** [comment, marked as answer]
Nothing wrong there. The thing is that with version 9, Tasmota expanded on the numbering system for the components configured for the gpio pins. The template you applied use the old numbering system, which was automatically converted to the new numbers. The extra 0 is that now the analog pin is configured as gpio17.

**Comment by elajoie** [comment]
cool, thanks!
